logic/src/commonMain/kotlin/com/wire/kalium/logic/data/client/ClientModel.kt

```diff
@@ -30,7 +30,9 @@ data class Client(
     val cookie: String?,
     val capabilities: Capabilities?,
     val model: String?
-)
+) {
+    val name by lazy { model ?: label ?: "" }
```

Comment:
I did move the name from AR to kalium, so we can have a unified way of displaying devices/clients across platforms.